lib/workers/repository/finalise/index.ts

```diff
@@ -2,13 +2,20 @@ import type { RenovateConfig } from '../../../config/types';
 import { platform } from '../../../modules/platform';
 import * as repositoryCache from '../../../util/cache/repository';
 import { clearRenovateRefs } from '../../../util/git';
+import { checkConfigMigrationBranch } from '../config-migration/branch';
+import { ensureConfigMigrationPr } from '../config-migration/pr';
 import { pruneStaleBranches } from './prune';
 
 // istanbul ignore next
 export async function finaliseRepo(
-  config: RenovateConfig,
+  config_: RenovateConfig,
   branchList: string[]
 ): Promise<void> {
+  let config = { ...config_ };
```

Comment:
why?

Reply:
config is const and im adding some props to it later on, so i guess it needed to be cloned

Reply:
similarly to what is done in `lib/workers/repository/init/config.ts`

Reply:
I don't think we need to copy here, so revert

Reply:
the const is only the variable reference, not the object itself 😉

Reply:
> I don't think we need to copy here, so revert

will do
that said, in this specific scope it is being passed around and needed to be modified.
guess its kinda off logically, regardless it will be reverted